UnitTests/Tests/TRWeatherUpdateCacheSpec.m

```diff
@@ -29,6 +29,14 @@ - (NSURL *)latestWeatherUpdateURL;
     [[NSFileManager defaultManager] removeItemAtURL:weatherUpdateURLForTesting() error:nil];
 };
 
+NSDictionary* (^weatherConditionsWithTemperature) (NSNumber*) = ^NSDictionary* (NSNumber *temp) {
```

Comment:
Hey, sorry, there should always be a space before the pointer, never one after. If we didn't do that elsewhere it slipped through code review. I'd write that line as 

```objc
NSDictionary *(^weatherConditionsWithTemperature) (NSNumber *) = ^NSDictionary *(NSNumber *temp) {
```